reactor-core/src/main/java/reactor/core/publisher/ContextPropagation.java

```diff
@@ -166,7 +166,7 @@ static <T, R> BiConsumer<T, SynchronousSink<R>> contextRestoreForHandle(BiConsum
 	}
 
 	static <T> SignalListener<T> contextRestoreForTap(final SignalListener<T> original, Supplier<Context> contextSupplier) {
-		if (propagateContextToThreadLocals || !ContextPropagation.isContextPropagationAvailable()) {
+		if (!ContextPropagation.isContextPropagationAvailable()) {
```

Comment:
Any reasons why this is changed?

Reply:
how about adding comment that this method is used in non-context restoring versions of tap operator

Reply:
yep, will do